File: src/main/java/com/starshootercity/abilities/incomplete/FlameParticles.java
package com.starshootercity.abilities.incomplete;

import com.starshootercity.abilities.Ability;
import net.kyori.adventure.key.Key;
import org.jetbrains.annotations.NotNull;

public class FlameParticles implements Ability {
    @Override
    public @NotNull Key getKey() {
        return Key.key("origins:flame_particles");
    }
}

File: core/src/main/kotlin/ru/turbovadim/abilities/FlameParticles.kt
package ru.turbovadim.abilities

import net.kyori.adventure.key.Key
import org.bukkit.Particle

class FlameParticles : ParticleAbility {
    override fun getKey(): Key {
        return Key.key("origins:flame_particles")
    }

    override val particle = Particle.FLAME

    override val frequency = 4
}
